Add "See all" link next to featured versions

bdsqqq opened the pull request:
A couple of friends and I who started using Modrinth recently didn't realize the "versions" tab existed right away. This PR aims to provide a way for users to easily find and access the "versions" tab from the "featured versions" section.

wafflecoffee reviewed (commented):
It's a good idea, but it looks quite out place right now.
![image](https://user-images.githubusercontent.com/60205699/175758315-f86056d1-fd81-4dcf-868d-b4c28b5ebaf4.png)
imo, it might be better as something like this:
![image](https://user-images.githubusercontent.com/60205699/175758467-f6e380e0-3245-491c-9f6c-ae55e7e3954e.png)
But if you or anyone else has any better ideas feel free to elaborate :)

bdsqqq commented:
Thanks for the feedback!
I really liked the approach you suggested of putting the link in the section header. I'll update the code for the pull request to reflect it as soon as I have some time :).

Minenash commented:
>  imo, it might be better as something like this:

that does look good, probably look a bit better if the () weren't apart of the link

wafflecoffee commented:
Is this still being worked on?

bdsqqq commented:
Changed the label to "(See all)", and moved it to after the "Featured versions" h3.

I kept the ( ) in the link because since the label is inside a `span class="links"` > `nuxt-link` > `span`, it would render "( see all)" eg: see all with an space after the parenthesis. I chose not to mess with the structure to keep this link consistent with the others I found in this file.

Also, to make the link render in the same line as the "Featured versions" `h3` I added `width: fit-content;` and `display: inline;` to `.card-header` 

On desktop:
![image](https://user-images.githubusercontent.com/37847523/186288182-b161fa1a-6912-408d-bb6b-f41bca72c52e.png)

On mobile: 
![image](https://user-images.githubusercontent.com/37847523/186288237-73ddc6c9-a654-4f67-b717-aac0d86f7b2b.png)

bdsqqq commented:
btw, @wafflecoffee sorry for the late response and VERY late update. I put this on my todo list and never got to it.
Hope this is still a welcome update as I think it would improve UX.
